index.html

```diff
@@ -312,14 +312,16 @@ <h3>
 listed here.
           </p>
           <ul>
+            <li>Test suites for all normative deliverables</li>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
+            <li>Extensions for binding multilingual resources for localized user interfaces.</li>
             <li>A Developer Guide consisting of one or more notes related to general implementation guidance and best practices for working with VCs, including but not limited to:
               <ul>
                 <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
                 <li>Guidance on Verifiable Credential Exchange over OpenID Connect</li>
```

Comment:
adding suggestions since there was support to add URLs to OIDC specs when we added GNAP:

```suggestion
                <li>Verifiable Credential Exchange over <a href="https://openid.net/specs/openid-connect-core-1_0.html">OpenID Connect</a>:</li>
                  <ul>
                    <li><a href="https://openid.net/specs/openid-connect-self-issued-v2-1_0.html">Self-Issued OpenID Provider v2</a></li>                          
                    <li><a href="https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html">OpenID Connect for Verifiable Presentations</a></li>
                    <li><a href="https://openid.net/specs/openid-connect-4-verifiable-credential-issuance-1_0.html">OpenID Connect for Verifiable Credential Issuance</a></li>
                  </ul>
```
(Happy to do a separate PR, but I think this fits here)

Reply:
@Sakurann if you are fine raising a separate PR for these changes, we can merge this PR now.